minecraft_servers/opsucht/manifest.json

```diff
@@ -35,5 +35,5 @@
     "country": "Germany",
     "country_code": "DE"
   },
-  "yt_trailer": "O7Zdr6ZygNo"
+  "yt_trailer": "watch?v=O7Zdr6ZygNo&t=2s"
```

Comment:
`yt_trailer` can only contain the "embed youtube ID". So everything was just right.